packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.mjml

```diff
@@ -67,7 +67,7 @@
         </mj-text>
 
         <mj-text css-class="footer-text">
-          <% if (locals.productName) { %>
+          <% if (locals.productName || locals.subscriptions.length > 0) { %>
```

Comment:
Ooo this breaks at least the "multiple products" subscription storybook layout. If you change this + the `.txt` version to `locals.subscriptions?.length > 0` I think that fixes the problem.

I stared at this for a few minutes but I think it's g2g with that fix. I was going to say I kind of wonder why we need this conditional at all, like in what scenarios we _don't_ want this link to show, but I imagine if a user has already cancelled then `subscriptions.length` is `0` and we don't send in a specific `productName` that they're subscribed to then we don't want to display this... guess we'll see as more templates are converted.

Reply:
Good catch! I'll make that change.

Yep, there are a couple of emails that have yet to be converted to the new template that don't have `productName` - mainly reminder emails to prompt the user to finish creating their Firefox account.

Reply:
Looks good to me. 🙂 Maybe don't do this here since it's another change and CI run, but when we get to the kind of subscription layout that doesn't render the link, let's add that to the subscription layout stories list too.